mvds.md

```diff
@@ -144,6 +145,15 @@ The record of the type `Type` SHOULD be retransmitted every time `Send Epoch` is
 1. <https://code.briarproject.org/briar/briar-spec/blob/master/protocols/BSP.md>
 2. <https://github.com/vacp2p/mvds>
 
+## Changelog
```

Comment:
Can we clearly call out type of change? e.g. breaking, feature, bugfix, docs

Reply:
@oskarth Not sure that makes sense, the diff between this usually contains multiple changes. The version number should denote whether it is a breaking change or not.

Reply:
disagree, a version number doesn't tell you anything about what has actually changed. That's what the changelog is for

Reply:
https://github.com/clojure/clojure/blob/master/changes.md#1-features-and-major-changes example

Reply:
@oskarth updated it a bit, did you want to have extensive documentation on everything changed or a basic overview.

Reply:
I guess it's fine for now, we can improve format in later releases (with the goal being to see at a glance what you should care about / if you can / want to upgrade)